ovssubnet/controller/kube/kube.go

```diff
@@ -72,6 +91,44 @@ func (c *FlowController) manageLocalIpam(ipnet *net.IPNet) error {
 	return nil
 }
 
+func (c *FlowController) SetupIptables(fw *firewalld.Interface, containerNetwork string) error {
+	if fw.IsRunning() {
+		err := fw.EnsureRule(firewalld.IPv4, "nat", "POSTROUTING", 0,
+			[]string{"-s", containerNetwork, "!", "-d", containerNetwork, "-j", "MASQUERADE"})
+		if err != nil {
+			return err
+		}
+		err = fw.EnsureRule(firewalld.IPv4, "filter", "INPUT", 0,
+			[]string{"-p", "udp", "-m", "multiport", "--dports", "4789", "-m", "comment", "--comment", "001 vxlan incoming", "-j", "ACCEPT"})
+		if err != nil {
+			return err
+		}
+		err = fw.EnsureRule(firewalld.IPv4, "filter", "INPUT", 0,
+			[]string{"-i", "tun0", "-m", "comment", "--comment", "traffic from docker for internet", "-j", "ACCEPT"})
```

Comment:
Assumption here is TUN=tun0 in the setup.sh, can we pass tun0 as arg to openshift-sdn-kube-subnet-setup.sh and setupIptables() that will ensure we use the same value in both places.